package.json

```diff
@@ -25,19 +25,23 @@
     "async": "2.4.1",
     "lodash": "~4.17.4",
     "oniyi-http-client": "1.1.1",
-    "oniyi-http-plugin-credentials": "0.1.0",
+    "oniyi-http-plugin-credentials": "github:futin/oniyi-http-plugin-credentials#d3aa547f11e9",
+    "oniyi-http-plugin-format-url-template": "bitbucket:gisag/oniyi-http-plugin-format-url-template#6d75931",
```

Comment:
lines 28 and 29 are waiting for version releases
**oniyi-http-plugin-credentials**: [here](https://github.com/benkroeger/oniyi-http-plugin-credentials/pull/1)
**oniyi-http-plugin-format-url-template**: [and here](https://bitbucket.org/gisag/oniyi-http-plugin-format-url-template/pull-requests/2/hotfix-falsy-method-fix/diff)